tests/0113-cooperative_rebalance.cpp

```diff
@@ -3265,9 +3370,18 @@ int main_0113_cooperative_rebalance_local(int argc, char **argv) {
 
   a_assign_rapid();
   p_lost_partitions_heartbeat_illegal_generation_test();
-  q_lost_partitions_illegal_generation_test(rd_false /*joingroup*/);
-  q_lost_partitions_illegal_generation_test(rd_true /*syncgroup*/);
-  r_lost_partitions_commit_illegal_generation_test_local();
+  if (test_consumer_group_protocol_classic()) {
+    /* These tests have no correspondence with
+     * the consumer group protocol "consumer" */
+    q_lost_partitions_illegal_generation_test(rd_false /*joingroup*/);
+    q_lost_partitions_illegal_generation_test(rd_true /*syncgroup*/);
+  }
+  if (test_consumer_group_protocol_classic()) {
+    r_lost_partitions_commit_illegal_generation_test_local();
+  } else if (0) {
+    /* FIXME: enable this once new errors are handled in OffsetCommit. */
+    t_consumer_group_consumer_retry_commit_on_fenced_member_epoch();
+  }
```

Comment:
I will enable this in my other opened PR.

Reply:
I checked it and I think we should remove this test, as now OffsetCommit returns STALE_MEMBER_EPOCH that is not retried automatically and FENCED _MEMBER_EPOCH is returned by the Heartbeat only